spec/source_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../classes/source'
+require_relative '../classes/item'
+describe Source do
+  describe '#new' do
+    before :each do
+      @source = Source.new('Online')
+    end
+    it 'should take two arguments and create a Source object' do
+      expect(@source).to be_an_instance_of Source
+    end
+    it 'should return a non-empty items array' do
+      item = Item.new('2022-04-21')
+      @source.items << item
+      expect(@source.items.empty?).to be false
+    end
```

Comment:
- [ ] A good suit test case for the `source` class would be testing the `add_item` method implemented inside the `source` class, so kindly implement a suit case for that.